prod-aos-rcar.yaml

```diff
@@ -0,0 +1,379 @@
+desc: "Xen-Troops development setup for Renesas RCAR Gen3 hardware"
```

Comment:
Maybe it is worth mentioning AOS in some way.

Reply:
done